CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.9.2-dev
```

Comment:
Nit: We usually don't add `-dev` to changelog entries; putting that in the pubspec is sufficient to ensure that users don't think this is a full release version.